src/components/KSelect/KSelect.vue

```diff
@@ -735,7 +735,7 @@ $kSelectInputHelpTextHeight: calc(var(--kui-line-height-20, $kui-line-height-20)
   .select-input {
     &.filtering-disabled {
       :deep(input:not([disabled])) {
-        caret-color: transparent;
+        caret-color: transparent !important;
```

Comment:
@adamdehaven I think this should fix the input caret flashing on mobile in portal but I haven't been able to reproduce the issue in the sandbox, even on the phone